queue_services/entity-emailer/src/entity_emailer/email_templates/RES-PAID.jinja2

```diff
@@ -0,0 +1,22 @@
+{% extends 'RES-common.jinja2' %}
+
+{% block page_title %}You have successfully filed your restoration with the BC Business Registry.{% endblock %}
+{% block title_message %}You have successfully filed your restoration with the BC Business Registry.{% endblock %}
+
+{% block attachments %}
+    <ul>
+        <li>Restoration Application</li>
+        <li>Receipt</li>
+    </ul>
+{% endblock %}
+
+{% block after_processing %}
+    <p>
+        Once your filing has been processed, the business will be
+        registered, and the company will receive the below outputs:
+    </p>
+    <ul>
+        <li>Notice of Articles</li>
+        <li>Restoration Certificate</li>
+    </ul>
```

Comment:
Should the attachment be called, "Restoration Certificate" as shown in the  Google sheet, or "Restoration Application"

Reply:
It should be `Certificate of Restoration`

Reply:
Fixed as requested

Reply:
Mihai mentioned in RC to follow the existing implementation for other emails. In that case we have to show the specific name. Ref: https://github.com/bcgov/lear/blob/main/queue_services/entity-emailer/src/entity_emailer/email_templates/BC-IA-PAID.html#L48

Reply:
Sure, np, I've added the following test parameters:

```python
@pytest.mark.parametrize(
    'restoration_type, attachment_name',
    [
        ('fullRestoration', 'Full Restoration Application'),
        ('limitedRestoration', 'Limited Restoration Application'),
        ('limitedRestorationExtension', 'Limited Restoration Extension Application'),
        ('limitedRestorationToFull', 'Conversion to Full Restoration Application'),
    ]
)
def test_paid_full_restoration_notification(session, restoration_type, attachment_name):
   # body of the test ...
```

Reply:
@vysakh-menon-aot FYI, I deliberately haven't changed the application file name from "Restoration Application.pdf" so to avoid lengthy filenames.  Do you think that's okay or would you prefer the filename match the full, descriptive name?  For example, `Conversion to Full Restoration Application.pdf`

Reply:
Although I realize that sometimes it's necessary, I personally hate it when auto-generated file names have spaces in them. Just a personal pet peeve.

Reply:
Agree on avoiding spaces in filenames. Mixedcase causes issues for folks too.
I'm ok with long names, if I look at Colin, etc, etc. the names hold no relation to the contents of the file.

On PPR we include registration numbers in the filename as lawyers will have lots of these downloaded.